### PR TITLE
checkbashisms: Remove.  Available in debian-devscripts

### PR DESCRIPTION
**Pull request by chkno**
This version was out of date & confusingly emitted perl warnings about
itself along with warnings about the shell scripts it checked.  The
version in debian-devscripts is more recent & doesn't have this problem.

###### Motivation for this change
Make it easier to find the real `checkbashisms` (it's included in `debian-devscripts`) by removing this old, broken copy.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lsix** [comment]
I wonder if adding a alias to `debian-scripts` could not make sense to “ease transition” (if there is any to be eased).

**Comment by chkno** [comment]
I don't think this package is widely used, or there would be more complaints or PRs about it emitting 31 lines of warnings about its own perl syntax before doing its real job and emitting similar-looking warnings about shell syntax in the shell scripts it is invoked on.

**Review by wamserma — APPROVED** [review]
LGTM.

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -69,7 +69,6 @@ mapAliases ({
   casperjs = throw "casperjs has been removed, it was abandoned by upstream and broken.";
   catfish = xfce.catfish; # added 2019-12-22
   cgmanager = throw "cgmanager was deprecated by lxc and therefore removed from nixpkgs."; # added 2020-06-05
-  checkbashism = checkbashisms; # added 2016-08-16
```
Comment (wamserma) [inline pkgs/top-level/aliases.nix:72]: Maybe put a note here that the standalone package has been removed.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please add an alias that throws that the package got removed.

**Comment by wamserma** [comment]
`pkgs/development/tools/misc/checkbashisms/default.nix` has been rewritten to extract the script from debian-devscripts, so this PR is obsolete.

**Pull request closed without merging**